src/couchapps/ReqMgr/updates/updaterequest.js

```diff
@@ -36,6 +36,19 @@ function(doc, req) {
     	return true;
 	}
 	
+	function updateTaskStepChain(chainType, prop, value) {
+		if (doc[chainType + "Chain"]) {
+			var numChain = doc[chainType + "Chain"];
```

Comment:
In javascript we don't have to check, if it doesn't contain the key it returns "undefined" which is create as false.
My editor automatically sets tab to 4 space but I am not sure why it is more than that. I will fix the indentation.